server/pbenchinacan/run-pbench-in-a-can

```diff
@@ -103,6 +110,12 @@ podman run \
 #
 # We do this in the Pbench Server container so that we get a known version of
 # openssl (the native one on the Jenkins executors appears to be ancient).
+#
+# Note that this command, if successful, dumps the status of the factorization
+# search to stderr which produces a bunch of junk in the output; thus, we
+# redirect stderr to stdout and then pipe it through sed to remove any complete
+# lines consisting solely of any combination of periods, plus signs, asterisks,
+# and hyphens.
```

Comment:
👍🏻

Reply:
I just cannot believe that there is no option to suppress it...but I couldn't find one.  (And, I sure as heck didn't want to just send _all_ of `stderr` to the bitbucket....)